src/operator/account/CancelAccountOverlay.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {FC, useContext} from 'react'
+import {
+  Overlay,
+  Gradients,
+  Alert,
+  ComponentColor,
+  IconFont,
+  ButtonBase,
+  ButtonShape,
+} from '@influxdata/clockface'
+import {AccountContext} from 'src/operator/context/account'
+
+const CancelAccountOverlay: FC = () => {
+  const {
+    account,
+    handleDeleteAccount,
+    setCancelOverlayVisible,
+    cancelOverlayVisible,
+  } = useContext(AccountContext)
+
+  const cancelAccount = () => {
+    if (account?.cancellable) {
+      try {
+        handleDeleteAccount()
+      } catch (e) {
+        setCancelOverlayVisible(false)
+      }
```

Comment:
Doesn't need to happen in this PR, but I think this may confuse folks since there's no notification anything went wrong - if deleteAccount fails, the overlay will just close. 

We may want a follow-on PR adding a pop-up notification indicating that the account couldn't be deleted (there is a `dispatch(notify)` function for that in the UI).